medical_pharmacy/models/medical_pharmacist.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016-2017 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import api, models, tools
+from odoo.modules import get_module_resource
+
+
+class MedicalPharmacist(models.Model):
+    _name = 'medical.pharmacist'
+    _description = 'Medical Pharmacist'
+    _inherit = 'medical.abstract.entity'
+
+    @api.model
+    def _create_vals(self, vals):
+        vals.update({
+            'is_company': False,
+            'customer': False,
+        })
+        return super(MedicalPharmacist, self)._create_vals(vals)
+
+    @api.model
+    def _get_default_image(self, vals):
+        res = super(MedicalPharmacist, self)._get_default_image(vals)
```

Comment:
Didn't we change the function _get_default_image to _allow_image_create?

Also, I wonder if we should not have a get_default_image implemented in medical_abstract_entity then only override a `get_default_image_path` when inheriting.


In the medical_abstract_entity.py class:

```
    @api.model_cr_context
    def _get_default_image_path(self):
        '''
        Override me !!!
        '''
        img_path = odoo.modules.get_module_resource(
            'base', 'static/src/img', 'avatar-for-undefined-medical-entity.png',
        )

    @api.model_cr_context
    def _get_default_image(self, vals):
        res = self._allow_image_create(vals)
        if not res:
            return res
        img_path = self._get_default_image_path()
        with open(img_path, 'r') as image:
            base64_image = image.read().encode('base64')
        return odoo.tools.image_resize_image_big(base64_image)
```